core/storage/user/gae_models.py

```diff
@@ -1244,12 +1317,17 @@ def apply_deletion_policy(cls, user_id):
         datastore_services.delete_multi(
             cls.query(cls.user_id == user_id).fetch(keys_only=True))
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a single shared instance."""
```

Comment:
Implementation does not match docstring :( Please, verify carefully, here and elsewhere. I think giving a clear insightful explanation for cases which are not "single unshared instance" should help with avoiding errors like this.

Reply:
I updated these docstrings. The "single unshared instance" is the default, so I have not elaborated on these, but I have elaborated on the others.